dts/bindings/usb/usb.yaml

```diff
@@ -26,6 +26,11 @@ properties:
                    via DT, USB controllers should use their maximum
                    hardware capability.
       generation: define
+      enum:
+         - "super-speed"
```

Comment:
Might as well add `super-speed-plus` to have the full range

Reply:
Does it make more sense to add when something uses it?

Reply:
I think we can add it later, but then it would be better to rearrange order of enums so adding new variant would not shift everything.

Reply:
Reordered.